assets/css/src/comments.css

```diff
@@ -147,19 +147,19 @@
 	font-family: var(--highlight-font-family);
 	font-stretch: condensed;
 	font-size: 85%;
-	color: #000;
+	color: var(--global-font-color);
 	line-height: 1.3;
 	text-decoration: none;
 	background: transparent;
-	border: 1px solid #c3c3c3;
+	border: 1px solid #ccc;
```

Comment:
I think for these `#ccc` borders we should also add a custom property, `--light-border-color`.

Reply:
Yes. I think `---border-color-light` would be more in line with what we have already.

Reply:
I think we should also rename `--global-border-color` to `--border-color-dark`. Ok for you @felixarntz?